index.js

```diff
@@ -121,6 +122,16 @@ function convertCrontabs() {
 class ServerlessLocalCrontabs {
   constructor(serverless, options) {
     this.serverless = serverless;
+
+    // Create schema for your properties. For reference use https://github.com/ajv-validator/ajv
+    this.serverless.configSchemaHandler.defineFunctionEventProperties('aws', 'schedule', {
```

Comment:
it should check if `this.serverless.configSchemaHandler` is defined first to avoid breaking for older Framework versions

Reply:
I will send PR directly then